### PR TITLE
Header fixes

### PR DESCRIPTION
**Pull request by NaominoDino**
I fixed some one the build errors. I’ll keep going on it!

### CODE REVIEW

File: src/video_core/renderer_vulkan/liverpool_to_vk.cpp [hunk 1]
```diff
@@ -222,6 +222,9 @@ vk::CompareOp DepthCompare(AmdGpu::DepthCompare comp) {
         return vk::CompareOp::eGreaterOrEqual;
     case AmdGpu::DepthCompare::Always:
         return vk::CompareOp::eAlways;
+    /// Missing AMG Gpu default statement
+    default:
+        throw std::invalid_argument("Unsupported DeepCompare");
```
Comment (raphaelthegreat) [inline src/video_core/renderer_vulkan/liverpool_to_vk.cpp]:
```suggestion
    default:
        UNREACHABLE_MSG("Unknown depth compare op {}", u32(comp));
```

File: src/core/libraries/kernel/libkernel.cpp [hunk 2]
```diff
@@ -295,8 +295,9 @@ int PS4_SYSV_ABI sceKernelUuidCreate(OrbisKernelUuid* orbisUuid) {
     return 0;
 }
 
-char* PS4_SYSV_ABI sceKernelGetFsSandboxRandomWord() {
-    char* path = "sys";
+/// Naomi Rework (Const)
```
Comment (raphaelthegreat) [inline src/core/libraries/kernel/libkernel.cpp:298]:
```suggestion
```

**Review by raphaelthegreat — CHANGES_REQUESTED** [review]

File: src/core/libraries/kernel/thread_management.cpp [hunk 3]
```diff
@@ -343,6 +345,18 @@ int PS4_SYSV_ABI scePthreadAttrSetstackaddr(ScePthreadAttr* attr, void* addr) {
     int result = pthread_attr_setstackaddr(&(*attr)->pth_attr, addr);
 
     return result == 0 ? SCE_OK : SCE_KERNEL_ERROR_EINVAL;
+}*/
+int PS4_SYSV_ABI scePthreadAttrSetstackaddr(ScePthreadAttr* attr, void* stack_addr, size_t stack_size) {
```
Comment (raphaelthegreat) [inline src/core/libraries/kernel/thread_management.cpp]: This change is wrong, please revert it

**Review by raphaelthegreat — CHANGES_REQUESTED** [review]

File: src/core/libraries/kernel/thread_management.cpp (hunk 3 above)
Comment (NaominoDino) [reply src/core/libraries/kernel/thread_management.cpp]: Done, reverted

File: src/sdl_window.cpp [hunk 4]
```diff
@@ -11,9 +11,18 @@
 #include "core/libraries/pad/pad.h"
 #include "input/controller.h"
 #include "sdl_window.h"
+#include <SDL3/SDL_mouse.h>
 
 namespace Frontend {
 
+void WindowSDL::CursorVisibility(bool cursor_visibility) {
```
Comment (raphaelthegreat) [inline src/sdl_window.cpp]:
```suggestion
void WindowSDL::SetCursorVisibility(bool cursor_visibility) {
```

**Review by raphaelthegreat — CHANGES_REQUESTED** [review]

**Pull request closed without merging**